### DISCUSSION TITLE
No measurements recorded during this period

### DISCUSSION THREAD

**Discussion by amit-saini-tx** [artilleryio/artillery#3364] [Artillery]
Hi,

I am encountering an issue where my tests running on AWS Fargate display the message: "No measurements recorded during this period." I've enabled debug logs, but I'm unable to identify any clear error messages or configuration issues. The strange part is that the same tests, with the same setup, are running perfectly fine locally.

Steps Taken:

- I've verified that AWS Fargate is working as expected by running a basic script, which executed successfully. This leads me to believe that the Fargate environment is not the issue.
- The tests began to fail only after I modified some files to parameterize the configuration and restructured the framework. I'm unsure whether these changes are causing the problem, but no error details are being recorded. But as I said earlier its working locally as expected with these changes.

For your reference, here are the test IDs:

AWS Fargate Test ID: t5kth_gfbqyptyw6dyjcgnazc3wd45d55j8_6ppz
Local Test ID: tbymf_c79g7dn6fggj8by9zt6xd36mjjayc_4raa
Basic Script on Fargate Test ID: trkq8_7c79y9nrngf6wckn3d5hr5749ypqx_p3y6

Could you please help investigate why these tests are failing on Fargate, despite working locally and in basic scripts?

Let me know if you need further details.

Thanks in advance!

**Comment by hassy** [comment]
@amit-saini-tx this is most likely to an unhandled exception in the test code which is causing the Fargate workers to crash. The best way to debug is to look at the logs in CloudWatch (via AWS Console -> ECS -> cluster you're using -> Tasks -> Logs). If you're reading from any files in your custom code (e.g. CSV files or the like), you also need to make sure to explicitly bundle them with `includeFile` - https://www.artillery.io/docs/reference/test-script#includefiles---explicitly-bundling-files-with-the-test

**Comment by JulianKingman** [comment]
FWIW, I encountered this problem when I was using fractional values for some fields, for example `rampTo` (e.g. `1.2` not `2`). Apparently doesn't work.
